package.json

```diff
@@ -44,7 +44,7 @@
     "jest-environment-jsdom": "^29.3.1",
     "markdownlint-cli2": "^0.5.1",
     "moment": "^2.29.4",
-    "obsidian": "0.14.6",
+    "obsidian": "^1.1.1",
```

Comment:
Please run `yarn` so that `yarn.lock` is updated to match this change. Thank you.

Reply:
Don't worry, I'll do it.